src/zepben/evolve/model/cim/iec61970/base/wires/power_transformer.py

```diff
@@ -294,11 +295,23 @@ class PowerTransformer(ConductingEquipment):
     result of the calculation S/Sn, where S = Load on Transformer (in VA), Sn = Transformer Nameplate Rating (in VA).
     """
 
-    def __init__(self, usage_points: List[UsagePoint] = None, equipment_containers: List[EquipmentContainer] = None,
-                 operational_restrictions: List[OperationalRestriction] = None, current_feeders: List[Feeder] = None, terminals: List[Terminal] = None,
+    asset_info: Optional[PowerTransformerInfo] = None
+    """
+    Override the [AssetInfo] as [PowerTransformerInfo].
```

Comment:
use backtick's (\`) instead of square brackets. also although painful but to do the pydoc properly you have to specify the full import, e.g: `zepben.evolve.model.cim.iec61968.assetinfo.power_transformer_info.PowerTransformerInfo`